react/components/ProgressBar/utils.ts

```diff
@@ -82,7 +82,7 @@ export function generatePackageProgressBarStates(
 }
 
 export function isOrderReadyToPickUp(packages: any) {
-  if (packages == null || packages.length === 0) return false
+  if (packages === null || packages.length === 0) return false
```

Comment:
you can leave the shallow equality here, I guess. This will check both null and undefined